src/SimpleWebRequest.ts

```diff
@@ -531,7 +531,7 @@ export class SimpleWebRequest<T> {
         }
 
         return {
-            url: this._url,
+            url: this._xhr.responseURL,
```

Comment:
This feature is only supported in certain browsers.  Please fall back with || this._url so that it maintains support across all environments.